G3D10/G3D-app.lib/source/GApp.cpp

```diff
@@ -1564,6 +1567,13 @@ void GApp::onSimulation(RealTime rdt, SimTime sdt, SimTime idt) {
     }
 
     if (scene()) { scene()->onSimulation(sdt); }
+
+    // If our renderer is (a subclass of) DefaultRenderer...
+    const shared_ptr<DefaultRenderer>& defaultRenderer = dynamic_pointer_cast<DefaultRenderer>(m_renderer);
+    if (notNull(defaultRenderer)) {
+        // ...update the dynamic diffuse GI
+        defaultRenderer->updateDiffuseGI(renderDevice, scene(), m_gbuffer, activeCamera());
+    }
```

Comment:
I believe the above lines are how DDGI is meant to update only if the `Renderer` happens to be a `DefaultRenderer` or subclass thereof.